app-service-proxy/test/headertest/header_test.go

```diff
@@ -0,0 +1,47 @@
+// SPDX-FileCopyrightText: (C) 2025-present Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package headertest
+
+import (
+	"fmt"
+	"strings"
+)
+
+func (s *TestSuite) TestAuthorizationHeader() {
+	for _, app := range s.DeployApps {
+		appID := *app.Id
+		appName := *app.Name
+		fmt.Println("Test header starting : ", appName)
```

Comment:
why use both fmt.Println() and s.T().Logf() ?

Reply:
removed the fmt log.